app/javascript/components/Login.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchUser } from '../redux/features/users/usersSlice';
+import '../../assets/stylesheets/navbar.css';
+
+const Login = () => {
```

Comment:
- [ ] When I try to login with a user that does not exist, it will  let me into the app with the nonexistent user:

Initial login:

![image](https://github.com/JuanDBta/booking-bites/assets/57421823/bfbc50b6-6c28-4c46-b3b2-0186f741c58d)

Inside the app with the nonexistent user:

![image](https://github.com/JuanDBta/booking-bites/assets/57421823/103636e4-836e-4bed-bbe0-8c6e31e83c9d)

The console:

![image](https://github.com/JuanDBta/booking-bites/assets/57421823/fe66e83d-0265-437f-a529-bedbe7318706)


## Solution

We need to remove this line:

![image](https://github.com/JuanDBta/booking-bites/assets/57421823/cb70d9e5-e7e2-4006-bed6-84e8e787c35b)

This is because we need the error to get passed to `Login.js`  so the navigate("/home") never gets reached in case of error.

Optionally, you can display the error in the console with `console.error(error.message)`.   You could even display the error in the frontend if you want.  

Result (tested):

![image](https://github.com/JuanDBta/booking-bites/assets/57421823/d1ee78aa-8366-49f2-89d1-e4c45c98469e)

If the user does not exist it won't let me in and will display the error in the console.

That way, only registered users will be able to get into the app, and your project will be perfect.  👍